### ISSUE TITLE
[Enhancement] Color of the app

### ISSUE THREAD

**Issue by YuriDevAT** [YurisCodingClub/html-tag-quiz#35]
Just a quick question regarding our design: will we have a dark/light toggle? 🙏
 Full confession: I am asking for very selfish reasons.
 I had to do some research last year because of my experience as a user with dark mode designs. I had noticed that unless an app’s content was the only way to access particular info, I would otherwise shy away from them. I just find them hard to focus on. But knowing how popular dark/light themes had become, I was just curious what makes a user prefer one over the other. 

At least according to research done in Germany in 2020, my experience is not that uncommon. While dark is certainly beneficial and recommended for some users, for the majority the so-called, ‘positive contrast polarity” was better with light mode. Here’s the article I saved last year: [Dark Mode vs. Light Mode: Which Is Better?](https://www.nngroup.com/articles/dark-mode/) Happy to explore more to see if this is still relevant ! And greatly appreciate your patience with me. 🙏

_Originally posted by @CorinaMurg in https://github.com/YurisCodingClub/html-tag-quiz/issues/18#issuecomment-1769515401_

**Comment by CorinaMurg** [comment]
Should I get started with a Figma mood board? Just so we have a visual about possible colors.

**Comment by YuriDevAT** [comment]
Love that idea. Thanks Corina for starting this!

**Comment by CorinaMurg** [comment]
I have been looking for inspiration for the app's color palette, and remembered someone mentioned this to me a while ago:
https://coolors.co/palettes/trending
(Sharing just in case you don't know it yet)

**Comment by YuriDevAT** [comment]
Yeah, there are so many of them out there, also "accessible color" ones. I think the first thing to decide would be if we have more than two colors. Color palettes always seem beautiful in combination but rarely when I use them (at least for me 🙊 )

Other ones (accessible Color Combos claimers):
https://toolness.github.io/accessible-color-matrix/
https://colors.dopely.top/contrast-checker/05024e-ada6a2
https://color.adobe.com/create/color-wheel

**Comment by CorinaMurg** [comment]
I am a bit overwhelmed by color palettes. I use them to compare different shades of the same color and maybe how two colors go together, but that's all.

I keep forgetting to confirm the original color combinations you suggested. I know there was a comment about (I think) blue/purple and yellow but I can't seem to find it. 

This is a very basic mood board. Testing several shades of blue/yellow/orange/purple. Some shades of gray as well if we decide to add a color to the background for each group of tags. Feel free to add/delete/change colors.
https://www.figma.com/file/FnY3XGLmYCSpMKoEDEqcPF/HTML-Tag-Quiz?type=design&node-id=0%3A1&mode=design&t=iGiYRQNDJm494V4O-1
